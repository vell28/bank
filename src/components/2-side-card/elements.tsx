import styled from 'styled-components';

export const TwoSideBox = styled.div`
  position: relative;
  width: 100%;
`;

export const CardRow = styled.div`
  margin-bottom: 13px;
  position: relative;
  &:last-child {
    margin-bottom: 0;
  }
  .error-field {
    background: ${(props) => props.theme.colors.white};
  }
`;

export const CardBox = styled.div`
  position: relative;
  padding: 24px 25px 18px 22px;
  border-radius: 10px;
  box-shadow: 0 2px 20px 0 rgba(0, 0, 0, 0.5);
  border: solid 1px #dfe1e4;
  background: ${(props) => props.theme.colors.white};
  width: 327px;
  height: 189px;
  z-index: 1;
`;

export const SecondCard = styled(CardBox)`
  position: absolute;
  right: 0;
  top: 19px;
  width: 262px;
  height: 151px;
  z-index: 0;
  padding: 24px 0 22px 0;
  ${CardRow} {
    width: 65px;
    position: absolute;
    right: 19px;
  }
`;

export const CardHeader = styled.div`
  display: flex;
  justify-content: space-between;
  margin-bottom: 8px;
`;

export const CardTitle = styled.div`
  color: ${(props) => props.theme.colors.black};
  font-size: 13px;
  font-weight: 500;
  letter-spacing: 0.98px;
  margin-bottom: 2px;
`;

export const CardLogo = styled.div`
  width: 54px;
  margin: 7px 0 0 0px;
`;

export const CardBody = styled.form`
  display: block;
`;

export const CardColumns = styled.div`
  display: flex;
  justify-content: space-between;
`;

export const CardColumn = styled<any>('div')`
  display: flex;
  flex-direction: column;
  align-items: baseline;
  width: ${(props) => (props.width ? `${props.width}px` : 'auto')};
`;

export const CardLabel = styled.label`
  color: #354d65;
  font-size: 8px;
  letter-spacing: normal;
  text-transform: uppercase;
  font-weight: 600;
  position: absolute;
  top: 0;
}
`;

export const CardHolderColumn = styled(CardColumn)`
  input {
    font-size: 10px;
    &::-webkit-input-placeholder {
      color: ${({ theme }) => theme.colors.gray};
      font-size: 17px;
      text-transform: none;
    }
    &:-ms-input-placeholder {
      color: ${({ theme }) => theme.colors.gray};
      font-size: 17px;
      text-transform: none;
    }
    &::-ms-input-placeholder {
      color: ${({ theme }) => theme.colors.gray};
      font-size: 17px;
      text-transform: none;
    }
    &::placeholder {
      color: ${({ theme }) => theme.colors.gray};
      font-size: 17px;
      text-transform: none;
    }
  }
`;

export const Line = styled.div`
  width: 100%;
  height: 33px;
  background-color: rgba(53, 77, 101, 0.12);
  margin-bottom: 10px;
`;
