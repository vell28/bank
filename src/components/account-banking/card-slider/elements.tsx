import styled from 'styled-components';

export const SliderBox = styled.div`
  position: relative;
  padding: 23px 25px 30px;
  max-width: 100%;
  overflow: hidden;

  .swiper-container {
    overflow: visible !important;
  }
  .arrow-btn {
    background: none !important;
    position: absolute;
    top: 73px;
    z-index: 10;
    cursor: pointer;
    transition: all 0.25s;
    outline: none;
    &.btn-next {
      right: 7px;
    }
    &.btn-prev {
      left: 10px;
    }
  }

  .account-card {
    position: relative;
    width: 210px;
    height: 121px;
    border-radius: 8px;
    background-color: #282929;
    box-shadow: 0 0 48px 0 rgba(0, 0, 0, 0.4);
    animation: 0.35s fadein;
    transition: all 0.5s;
    opacity: 0;
    background-image: url(/img/Card.png);
    background-repeat: no-repeat;
    background-position: 50%;
    background-size: contain;
    cursor: pointer;
  }

  .add-card {
    position: relative;
    width: 210px;
    height: 121px;
    border-radius: 8px;
    background-color: ${(props) => props.theme.colors.darkBlue};
    box-shadow: 0 0 48px 0 rgba(0, 0, 0, 0.4);
    animation: 0.35s fadein;
    transition: all 0.5s;
    opacity: 0;
    background-repeat: no-repeat;
    background-position: 50%;
    background-size: contain;
    cursor: pointer;
  }

  .swiper-slide:not(.swiper-slide-active) .account-card {
    transform: scale(0.71);
  }

  .swiper-slide:not(.swiper-slide-active) .add-card {
    transform: scale(0.71);
  }

  .swiper-slide {
    &.swiper-slide-prev {
      z-index: 9;
      position: relative;
      .account-card {
        left: -20%;
        opacity: 1;
      }
      .add-card {
        left: -20%;
        opacity: 1;
      }
    }

    &.swiper-slide-active {
      z-index: 10;
      .account-card {
        position: relative;
        left: 50%;
        transform: translateX(-50%);
        opacity: 1;
      }
      .add-card {
        position: relative;
        left: 50%;
        transform: translateX(-50%);
        opacity: 1;
      }
    }

    &.swiper-slide-next {
      .account-card {
        right: 25%;
        opacity: 1;
      }
      .add-card {
        right: 25%;
        opacity: 1;
      }
    }
  }

  .swiper-pagination {
    bottom: -30px !important;

    .swiper-pagination-bullet {
      width: 5px;
      height: 5px;
      margin: 0 2.5px;
      display: inline-block;
      border-radius: 50%;
      background-color: ${(props) => props.theme.colors.white};
      opacity: 0.4;
      cursor: pointer;
      transition: all 0.25s;
      &.swiper-pagination-bullet-active {
        opacity: 1;
      }
    }
  }
`;
