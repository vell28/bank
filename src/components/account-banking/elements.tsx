import styled from 'styled-components';

export const AccountBlock = styled.div`
  margin-top: 32px;
  margin-bottom: 26px;
  &:first-child {
    margin-top: 0 !important;
  }
`;

export const AccountTabs = styled.div`
  margin: 0 0 20px;
  display: flex;
  align-items: center;
  justify-content: space-around;
`;

export const SliderBox = styled.div`
  position: relative;
  margin-bottom: 30px;
  min-width: 485px;
  left: -25px;
`;

export const DetailBox = styled<any>('div')`
  height: ${(props) => (props.isShown ? '180px' : 'auto')};
`;

export const CardsBlock = styled(AccountBlock)`
  h2,
  .h2 {
    margin-bottom: 0;
  }
`;
