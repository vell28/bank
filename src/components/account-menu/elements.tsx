import styled, { css, keyframes } from 'styled-components';

import { Link } from 'react-router-dom';

export const AccountMenuBox = styled<any>('div')`
  width: 100%;
  max-height: ${(props) => (props.maxHeight ? `${props.maxHeight}px` : 'calc(100vh - 200px)')};
  padding: 0 15px;

  ${(props) => props['data-overflow'] && 'overflow: auto;'};
`;

export const widthKey = keyframes`
0%{width: 0;}
100%{width: 360px}
`;

const animationWidth = css`
  animation: ${widthKey} 0.6s ease-out;
`;

export const DropDown = styled.div`
  width: 100%;
  margin-top: 27px;
  text-align: center;
  span {
    letter-spacing: 0.9px;
    padding-right: 2px;
    font-size: 16px;
    color: ${(props) => props.theme.colors.blue};
    background-color: transparent;
    cursor: pointer;
    &:hover {
      text-decoration: underline;
    }
  }
`;

export const AccountBox = styled.div`
  position: relative;
  &:first-child {
    margin-top: 0 !important;
  }
  margin-top: 22px;
`;

export const AccountType = styled<any>('div')`
  display: flex;
  align-items: center;
  justify-content: space-between;
  font-size: 16px;
  letter-spacing: 0.9px;
  margin-top: ${(props) => props.top || 0}px;
  padding-left: ${(props) => props.left || 0}px;
  padding-bottom: 4px;
  span {
    overflow: hidden;
    text-overflow: ellipsis;
    &:first-child {
      width: 100%;
    }
    &:last-child {
      width: 80px;
      text-transform: lowercase;
    }
  }
`;

export const AccountBlock = styled.div``;

export const AccountCurrenciesBox = styled.div`
  width: 100%;
  cursor: pointer;
  margin: 10px 0 0;
  padding: 0;
  position: relative;
  transition: all 0.2s ease-out;
  &:hover {
    text-shadow: 1px 0 black;
  }

  &:hover a {
    text-shadow: 1px 0 white;
  }

  &.active-account a {
    text-shadow: 1px 0 white;
    &:after {
      content: '';
      position: absolute;
      top: 50%;
      left: 100%;
      width: 360px;
      ${animationWidth};
      will-change: width;
      margin-left: 8px;
      border-bottom: 1px solid ${(props) => props.theme.colors.white};
    }
  }

  &.active-account:hover {
    margin-left: 0;
  }
`;

export const AccountCurrencies = styled.div`
  position: relative;
  font-size: 18px;
  text-decoration: none;
}
`;

export const LinkCurrencies = styled(Link)`
  position: relative;
  font-size: 18px;
  text-decoration: none;
  transition: all 0.25s;
`;
