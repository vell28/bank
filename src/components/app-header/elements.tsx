import styled from 'styled-components';
import { Link } from 'react-router-dom';

export const Header = styled.div`
  padding: 22px 0 12px;
  width: 1220px;
  margin: auto;
`;

export const HeaderWrap = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-between;
`;

export const HeaderLogo = styled.div`
  img {
    width: 190px;
    height: 38px;
    vertical-align: baseline;
  }
`;

export const HeaderNav = styled.nav`
  margin-bottom: 5px;
`;

export const HeaderMenu = styled.ul`
  list-style-type: none;
  margin: 0;
  padding: 0;
`;

export const MenuItem = styled.li`
  display: inline-block;
`;

export const HeaderLngSelector = styled.div`
  margin-bottom: 3px;
  margin-left: 10px;
  font-size: 14px;
  color: ${(props) => props.theme.colors.white};
`;

export const LinkChildren = styled(Link)`
`;
