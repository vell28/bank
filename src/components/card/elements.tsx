import styled from 'styled-components';

export const CardBox = styled.div`
  padding: 21px 22px 16px;
  border: 1px solid #dfe1e4;
  box-shadow: 0 3px 7px 0 hsla(0, 0%, 70%, 0.26);
  border-radius: 10px;
  width: 100%;
  max-width: 350px;
`;

export const CardHeader = styled.div`
  display: flex;
  justify-content: space-between;
`;

export const CardTitle = styled.h3`
  margin: 12px 0 0 0;
  color: ${(props) => props.theme.colors.black};
  font-size: 13px;
  font-weight: 500;
  letter-spacing: 0.98px;
`;

export const CardLogo = styled.div`
  width: 54px;
  margin: 7px 0 0 0px;

  img {
    height: 42px;
    object-fit: cover;
  }
`;

export const CardBody = styled.form`
  display: block;
`;
export const CardRow = styled.div`
  margin-bottom: 8px;
  &:last-child {
    margin-bottom: 0;
  }
`;

export const CardColumns = styled.div`
  display: flex;
  justify-content: space-between;
`;

export const CardColumn = styled<any>('div')`
  display: flex;
  align-items: center;
  width: ${(props) => (props.isFullSize ? '100%' : '43%')};
`;

export const CardLabel = styled.label`
  margin-right: 5px;
  color: ${(props) => props.theme.colors.black};
  font-size: 9px;
  letter-spacing: .68px;
  text-transform: uppercase;
}
`;
