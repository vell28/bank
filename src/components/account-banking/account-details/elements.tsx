import styled from 'styled-components';

import { ActionBtn } from '../../common/button/action/elements';

export const AccountDetailBox = styled.div`
  position: relative;
  padding: 0 24px;
  font-family: AvenirNext;
  letter-spacing: 1.14px;
`;

export const BankId = styled.div`
  letter-spacing: 1.14px;
  font-size: 16px;
  color: ${(props) => props.theme.colors.black};
  margin-top: 10px;
`;

export const BankDetail = styled.div`
  margin: 17px 0 15px;
  display: flex;
`;

export const BankDetailBox = styled.div`
  width: 50%;
  &:last-child {
    margin-left: 40px;
  }
`;

export const FieldTitle = styled.div`
  font-size: 14px;
  letter-spacing: -0.09px;
  color: ${(props) => props.theme.colors.gray3};
`;

export const FieldValue = styled(FieldTitle)`
  color: ${(props) => props.theme.colors.black};
`;

export const ActionBox = styled.div`
  margin: 0 -24px 0 -24px;
  padding: 12px 24px 0 24px;
  border-top: 1px solid ${(props) => props.theme.colors.gray4};
  display: flex;
  justify-content: center;
`;

export const DetailActionBtn = styled(ActionBtn)`
  margin: 0 15px;
`;
