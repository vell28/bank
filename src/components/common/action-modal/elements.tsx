import styled from 'styled-components';

import { ActionBtn } from '../button/action/elements';

export const PopupOverlay = styled<any>('div')`
  backdrop-filter: blur(20px);
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 99;
  display: flex;
  flex-direction: column;
  align-items: center;
  overflow: hidden;
  background-color: rgba(1, 31, 61, 0.83);
  cursor: pointer;
  transition: transform 0.2s ease;
  visibility: visible;
  opacity: 1;
  user-select: none;
  -webkit-overflow-scrolling: touch;
  ${(props) =>
    props.hidden
    && `
    visibility: hidden;
    opacity: 0;
    z-index: -1;
  `};
`;

export const PopupBox = styled<any>('div')`
  position: relative;
  flex-shrink: 0;
  margin-top: auto;
  margin-bottom: auto;
  background: ${(props) => props.theme.colors.white};
  transform: scale(1);
  cursor: auto;
  opacity: 1;
  transition: all 0.3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
  width: 320px;
  border-radius: 17px;

  ${(props) =>
    props.hidden
    && `
    width: 60%;
    border-radius: 4px;
    transform: scale(0.8);
  `};
`;

export const PopupInner = styled.div`
  padding: 0 43px;
  padding-top: 80px;
  padding-bottom: 39px;
  color: ${(props) => props.theme.colors.black};
`;

export const PopupTitle = styled.h2`
  color: ${(props) => props.theme.colors.black};
  margin-bottom: 56px;
  font-size: 17px;
  font-weight: 600;
  letter-spacing: 0.5px;
`;

export const PopupDescription = styled.div`
  margin-bottom: 61px;
  font-size: 15px;
  letter-spacing: 0.5px;
  text-align: center;
`;

export const ActionBLock = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-around;
`;

export const PopupAction = styled<any>(ActionBtn)`
  color: ${(props) => (props.isReject ? props.theme.colors.red : props.theme.colors.blue)};
`;
