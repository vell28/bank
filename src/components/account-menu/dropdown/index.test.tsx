import React from 'react';

import { ThemeProvider } from 'styled-components';
import { enzymeFind } from 'styled-components/test-utils';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import { mainDarkTheme } from 'modules/themes';

import { accountsData } from 'models/organizations/mock';
import { AccountMenuDropdown } from '.';
import { AccountCurrencies } from '../elements';

describe('<AccountMenuDropdown /> tests', () => {
  test('should success render, should been unchecked', () => {
    const onDone = jest.fn();
    const component: any = mount(
      <ThemeProvider theme={mainDarkTheme}>
        <AccountMenuDropdown
          onDone={onDone}
          personalAccounts={[accountsData[0]]}
          businessAccounts={[accountsData[1]]}
        />
      </ThemeProvider>,
    );
    const checkbox = enzymeFind(component, AccountCurrencies);
    expect(checkbox.length).toEqual(2);
    expect(toJson(component)).toMatchSnapshot();
  });
});
