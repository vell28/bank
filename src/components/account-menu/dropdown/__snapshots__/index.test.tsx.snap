// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountMenuDropdown /> tests should success render, should been unchecked 1`] = `
<ThemeProvider
  theme={
    Object {
      "breakpoints": Array [
        "576px",
        "768px",
        "992px",
        "1200px",
      ],
      "colors": Object {
        "alto": "#dedede",
        "asparagus": "#77a45c",
        "black": "#000000",
        "blue": "#008cff",
        "blueLight": "#70cde4",
        "borderLight": "#cad3db",
        "dark": "#030303",
        "darkBlue": "#011f3d",
        "darkBlue2": "#132F4B",
        "elephant": "#12304b",
        "feijoa": "#93d77e",
        "gray": "#a5adb8",
        "gray10": "#cad3db",
        "gray11": "#959fac",
        "gray2": "#a7a9ac",
        "gray3": "#a1aab6",
        "gray4": "#e8e8e8",
        "gray5": "#cbcbcb",
        "gray6": "#eaeaea",
        "gray7": "#acacac",
        "gray9": "#9faab7",
        "grayHover": "#00000008",
        "grayLight": "#cdd0d5",
        "green": "#7ed321",
        "green1": "#5ba00f",
        "green2": "#4dd964",
        "green3": "#77a45c",
        "green4": "#b1de6b",
        "greenLight": "#d9f9b6",
        "light": "#d5dae0",
        "malibu": "#53b8ff",
        "navy": "#01172d",
        "orange": "#e4be79",
        "persimmon": "#ff5353",
        "purple": "#a479cb",
        "red": "#e10606",
        "red2": "#ff2e7b",
        "red3": "#ea4848",
        "rose": "#d08484",
        "slate": "#596778",
        "submarine": "#b6bcc4",
        "turquoise": "#79cbcb",
        "white": "#ffffff",
        "white2": "#dddddd",
      },
      "dimension": Object {},
      "fonts": Object {
        "faFontsPath": "/css/webfonts",
        "fontFamily": "AvenirNext, Arial",
      },
      "name": "main-dark-theme",
      "size": "default",
      "typography": Object {
        "bodyBackground": "#011f3d",
        "bodyFontColor": "#ffffff",
        "bodyFontFamily": "AvenirNext, Arial",
        "globalFontSize": "1.6rem",
        "globalLineHeight": 1.5,
        "globalWeightNormal": 400,
        "headersFontColor": "#ffffff",
        "headersFontFamily": "AvenirNext, Arial",
        "headersFontWeight": 700,
      },
    }
  }
>
  <AccountMenuDropdown
    businessAccounts={
      Array [
        Object {
          "accountName": "MT98PAPY36836000002676370004519",
          "accountType": "JURIDICAL",
          "accruedInterest": null,
          "availableBalance": 1897.61,
          "availableOperations": Array [],
          "balance": 0,
          "balanceHold": 0,
          "balances": Array [
            Object {
              "availableBalance": 1897.61,
              "balance": 1030.05,
              "balanceHold": 0,
              "currency": "EUR",
            },
          ],
          "bonusProgramActivate": false,
          "clientId": "6449",
          "color": "DEFAULT",
          "createdAt": "2019-11-24T14:59:55",
          "currency": Object {
            "code": "EUR",
          },
          "id": "PPY6666",
          "number": "MT98PAPY36836000002676370004519",
          "status": "ACTIVE",
        },
      ]
    }
    onDone={[MockFunction]}
    personalAccounts={
      Array [
        Object {
          "accountName": "MT98PAPY36836000002676370004519",
          "accountType": "PHYSICAL",
          "accruedInterest": null,
          "availableBalance": 1897.61,
          "availableOperations": Array [],
          "balance": 0,
          "balanceHold": 0,
          "balances": Array [
            Object {
              "availableBalance": 1897.61,
              "balance": 1030.05,
              "balanceHold": 0,
              "currency": "EUR",
            },
          ],
          "bonusProgramActivate": false,
          "clientId": "6449",
          "color": "DEFAULT",
          "createdAt": "2019-11-24T14:59:55",
          "currency": Object {
            "code": "EUR",
          },
          "id": "PPY5107",
          "number": "MT98PAPY36836000002676370004519",
          "status": "ACTIVE",
        },
      ]
    }
  >
    <styled.div
      data-overflow="true"
    >
      <div
        className="sc-AxjAm hEOMRA"
        data-overflow="true"
      >
        <styled.div>
          <div
            className="sc-AxiKw xorXm"
          >
            <styled.div
              left={31}
            >
              <div
                className="sc-AxhCb cvqgRz"
              >
                <span>
                  personal
                </span>
              </div>
            </styled.div>
            <styled.div
              key="PPY5107EUR"
            >
              <div
                className="sc-AxgMl ivHztB"
              >
                <styled.div>
                  <div
                    className="sc-AxheI hRvQHy"
                  >
                    <CheckBox
                      isChecked={false}
                      onCheck={[Function]}
                    >
                      <styled.div
                        isChecked={false}
                        onClick={[Function]}
                        size={20}
                      >
                        <div
                          className="sc-fzozJi ihBdPx"
                          onClick={[Function]}
                          size={20}
                        >
                          <styled.span
                            size={20}
                          >
                            <span
                              className="sc-AxmLO jGZvWA"
                              size={20}
                            />
                          </styled.span>
                          <styled.div>
                            <div
                              className="sc-fzoLsD kOiJJR"
                            >
                              €
                              1030.05
                            </div>
                          </styled.div>
                        </div>
                      </styled.div>
                    </CheckBox>
                  </div>
                </styled.div>
              </div>
            </styled.div>
            <styled.div
              left={31}
              top={27}
            >
              <div
                className="sc-AxhCb eYQlxe"
              >
                <span>
                  business
                </span>
              </div>
            </styled.div>
            <styled.div
              key="PPY6666EUR"
            >
              <div
                className="sc-AxgMl ivHztB"
              >
                <styled.div>
                  <div
                    className="sc-AxheI hRvQHy"
                  >
                    <CheckBox
                      isChecked={false}
                      onCheck={[Function]}
                    >
                      <styled.div
                        isChecked={false}
                        onClick={[Function]}
                        size={20}
                      >
                        <div
                          className="sc-fzozJi ihBdPx"
                          onClick={[Function]}
                          size={20}
                        >
                          <styled.span
                            size={20}
                          >
                            <span
                              className="sc-AxmLO jGZvWA"
                              size={20}
                            />
                          </styled.span>
                          <styled.div>
                            <div
                              className="sc-fzoLsD kOiJJR"
                            >
                              €
                              1030.05
                            </div>
                          </styled.div>
                        </div>
                      </styled.div>
                    </CheckBox>
                  </div>
                </styled.div>
              </div>
            </styled.div>
          </div>
        </styled.div>
        <styled.div
          onClick={[Function]}
        >
          <div
            className="sc-AxirZ ixoHnv"
            onClick={[Function]}
          >
            <span>
              done
            </span>
          </div>
        </styled.div>
      </div>
    </styled.div>
  </AccountMenuDropdown>
</ThemeProvider>
`;
